rasa/core/policies/memoization.py

```diff
@@ -236,6 +236,16 @@ def predict_action_probabilities(
 
         return result
 
+    def format_tracker_states(self, states):
+        # Format tracker states to human readable format on debug log
+        formatted_states = ''
+        for index, state in enumerate(states):
+            if bool(state):
+                formatted_states += '\n' + '[state ' + str(index) + '] ' \
+                    + 'previous action: ' + str(state['prev_action']['action_name']) \
+                    + ' | user intent: ' + str(state['user']['intent'])
```

Comment:
I think using f-strings here would make it more readable. You can use parenthesis to use multiline f-strings.